frontend/public/locales/en/translation.json

```
@@ -1938,8 +1945,10 @@
   "openshift.cluster.manager": "OpenShift Cluster Manager",
   "OpenStack clouds.yaml": "OpenStack clouds.yaml",
   "Operator": "Operator",
+  "Operator policy": "Operator policy",
   "Operator required": "Operator required",
   "Operator upgrade required": "Operator upgrade required",
+  "operatorpolicy.version.label.helper": "Versions is a list of non-empty strings that specifies which installed versions are compliant when set to `inform` mode, and which installPlans are approved when when set to `enforce` mode",
```

Reverting back to single quotes since monospace is not supported

```suggestion
  "operatorpolicy.version.label.helper": "Versions is a list of non-empty strings that specifies which installed versions are compliant when set to `inform` mode, and which installPlans are approved when when set to 'enforce' mode",
```